src/components/date-picker-month-header/date-picker-month-header.scss

```diff
@@ -117,22 +117,16 @@
 }
 
 .year--suffix {
-  @apply w-16 text-left;
+  @apply text-left;
 }
 
 .year-wrap {
-  @apply relative;
+  @apply w-16 relative;
 }
 
 .suffix {
-  @apply absolute
-    top-0
+  @apply top-0
     left-0
-    w-16
     whitespace-nowrap
-    text-left;
-}
-
-.suffix__invisible {
-  @apply invisible;
+    text-right;
```

Comment:
Should this be `text-right` or something that flips in RTL?

Reply:
same here,  it doesn't need to flip. The order has to respect the direction.